src/js/events.js

```diff
@@ -39,6 +39,19 @@ define(function (require, exports, module) {
         },
         document: {
             history: {
+                unifiedHistory: {
```

Comment:
as mentioned in the PR description, eventually when ALL the history events get migrated to the new system, then "unifiedHistory" will just get bumped up a level and we'll remove the other three.